modules/ROOT/pages/quickstart-functions.adoc

```diff
@@ -40,36 +41,49 @@ $ docker login -u <username> -p <password>
 
 . Many `faas` CLI commands require a `--repository` argument.
 +
-You can set the `FAAS_REPOSITORY` environment variable so that you can omit this parameter when using the CLI:
+You can set the `FAAS_REGISTRY` environment variable so that you can omit this parameter when using the CLI:
 +
 
 [source,terminal]
 ----
-$ export FAAS_REPOSITORY=docker.io/<username>/<repository_name>
+$ export FAAS_REGISTRY=docker.io/<username>
 ----
 
 == Create a Functions project
 
-. Create a project directory:
+. Create a project directory, with a function and runtime:
 +
 
 [source,terminal]
 ----
-$ mkdir fn.example.io
+$ mkdir <directory_name> && faas init <function_name> -l node
 ----
 
-. Navigate to the new directory:
+.Example output
+[source,termina]
+----
+Project path: /home/<user>/<directory_name>
+Function name: <function_name>
+Runtime: node
+Trigger: http
+----
+// Question: can the trigger be set to anything else?
```

Comment:
By default the trigger is `http`, but you can use the `-t` flag when initializing to create a function that responds to CloudEvents. E.g.

```
faas init <directory_name> -l node -t events
```